lib/active_set/column_instruction.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require_relative './attribute_instruction'
+
+class ActiveSet
+  class ColumnInstruction
+    def initialize(instructions_hash, item)
+      @instructions_hash = instructions_hash.symbolize_keys
+      @item = item
+    end
+
+    def key
```

Comment:
The key for a column instruction will be either a set `:key` entry, an I18n translated value for the attribute, or a titleized string.